lib/persistence/redis.js

```diff
@@ -252,6 +260,23 @@ RedisPersistence.prototype.storeSubscriptions = function(client, cb) {
   });
 
   var op = this._client.multi()
+    .get(clientSubKey, function(err, currentSubs){
```

Comment:
I'd have preferred to write this with Promises, to merge two `publishes`, however it won't happen in a single multi, since we need first to get current subscriptions before updating them